andela/metis-ah
# 161050894 Get featured articles

**What does this PR do?**
Implement get featured articles for the current week

**Description of Task to be completed?**

write tests for the endpoint
validate `limit` query params
determine the beginning of a week
get the following endpoint working: GET` api/v1/articles/featured`

**How should this be manually tested?**

Clone the repository
Check out the `ft-get-featured-articles-161050894` branch
Run `npm install` on your local machine to install required Dependencies
Run database migrations with `npm run migrate`
Run `npm test`
**Using Postman**
Run `npm run seed_data` after cloning and running database migrations
Start the server using `npm start`

Go to GET `localhost:8000/api/v1/articles/featured`with postman


What are the relevant pivotal tracker stories?
#161050894
